client/app/bundles/course/survey/pages/ResponseIndex/index.jsx

```diff
@@ -23,7 +23,9 @@ import surveyTranslations from 'course/survey/translations';
 import LoadingIndicator from 'lib/components/LoadingIndicator';
 import UnsubmitButton from 'course/survey/containers/UnsubmitButton';
 import { surveyShape, responseShape } from 'course/survey/propTypes';
+import { useTheme } from '@mui/material/styles';
 import RemindButton from './RemindButton';
+import { workflowStates } from '../../../assessment/submission/constants';
```

Comment:
the workflowStates constant belong to submission related constants. Perhaps, you could create similar constants just for survey? Even though the workflow states are the same, let's separate the constants.